StorageGUI.cs

```diff
@@ -530,15 +530,13 @@ private static bool TryDepositAll(bool quickStack)
 
 			bool filter(Item item) => !item.IsAir && !item.favorited && (!quickStack || heart.HasItem(item, true));
 			var items = new List<Item>();
-
-			//NEED to ignore mouse item (slot 58), otherwise the item gets cloned for whatever reason
-			// - absoluteAquarian
-			for (int k = 10; k < 58; k++)
+			
+			for (int k = 10; k < 54; k++)
 			{
 				Item item = player.inventory[k];
-				if (filter(item))
-					items.Add(item);
-			}
+                if (filter(item))
```

Comment:
Please revert these lines with space indents back to lines with tab indents.

Reply:
Done!